### PR TITLE
early 2023 projects met with Christina

### PR DESCRIPTION
**Pull request by ChristinaLK**

### CODE REVIEW

**Comment by brianhlin** [comment]
@jstathas this adds a new campus organization "Montgomery College". Is that ok?

**Comment by jstathas** [comment]
Brian,

Yes - "Montgomery College" is good to add.

Janet


Janet Stathas (she/her)

Project Manager

Morgridge Institute for Research

PATh<https://path-cc.io/>

Email: ***@***.***

Mobile: 608.345.9605

________________________________

File: projects/MontgomeryCollege_Dillman.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Description: workforce training, 2-5 day workshops every couple of months to get bench biologists comfortable with various bioinformatics pipelines on the command line.
```
Comment (brianhlin) [inline projects/MontgomeryCollege_Dillman.yaml:1]:
```suggestion
Description: >-
  workforce training, 2-5 day workshops every couple of months to get
  bench biologists comfortable with various bioinformatics pipelines
  on the command line.
```

File: projects/UTK_Luettgau.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Description: Piloting the National Science Data Fabric, A Platform Agnostic Testbed for Democratizing Data Delivery
```
Comment (brianhlin) [inline projects/UTK_Luettgau.yaml:1]:
```suggestion
Description: >-
  Piloting the National Science Data Fabric, A Platform Agnostic
  Testbed for Democratizing Data Delivery
```

**Review by brianhlin — APPROVED** [review]
LGTM with minor edits for project description readability (https://yaml-multiline.info/)

**Pull request merged**